cli/cmd/org/edit.go

```diff
@@ -57,9 +57,14 @@ func EditCmd(ch *cmdutil.Helper) *cobra.Command {
 				Name: org.Name,
 			}
 
+			flagsSet := cmd.Flags().Changed("display-name") || cmd.Flags().Changed("description") || cmd.Flags().Changed("billing-email")
+			if !flagsSet {
+				return fmt.Errorf("at least one flag must be set")
+			}
+
 			if cmd.Flags().Changed("display-name") {
 				req.DisplayName = &displayName
-			} else if ch.Interactive {
+			} else if ch.Interactive && !flagsSet {
```

Comment:
Can't you remove all these prompt cases now that you return early when `!flagsSet` on line 61?